packages/web3-eth-contract/test/integration/contract_deploy.test.ts

```diff
@@ -36,14 +37,14 @@ describe('contract', () => {
 		let acc: { address: string; privateKey: string };
 		let pkAccount: { address: string; privateKey: string };
 		beforeAll(async () => {
-			pkAccount = await createNewAccount({ refill: true });
-			acc = await createTempAccount();
 			deployOptions = {
 				data: GreeterBytecode,
 				arguments: ['My Greeting'],
 			};
 		});
```

Comment:
This entire `beforeAll` can be removed if `deployOptions` is set upon initialization

Reply:
done